binding/react/README.md

```diff
@@ -178,3 +178,9 @@ function VoiceWidget(props) {
     inferenceEventHandler
   );
 ```
+
```

Comment:
i know `vue` binding is not our best web binding, but still it can use some love here as well :)

Reply:
I knew I was missing one :)